docs/usermanual.md

```diff
@@ -450,6 +452,40 @@ a transformation expression in the IoTAgent). In this cases, when a reverse expr
 bidirectional attribute is modified, the IoTAgent sends a command to the original device, with the name defined in the
 reverse expression attribute and the ID of the device (see Commands Syntax, just above).
 
+#### Commands transformations
+
+It is possible to use expressions to transform commands, in the same way that other attributes could do it, that is
+adding `expression` to command definition. This way a command could be defined like:
+
+```json
+{
+    "name": "reset",
+    "type": "command",
+    "expression": "{ set: 0}"
+}
+```
+
+and when command will be executed the command value will be the result of apply value to defined expression. Followig
+the example case the command will be:
+
+```json
+{
+    "set": 0
+}
+```
```

Comment:
Should be in UL format? Something like `set|0` (unsure)

Reply:
You are right... this is the right one: https://github.com/telefonicaid/iotagent-ul/pull/550#discussion_r838661612

(I was reviewing the two PRs in parallel and I put the comment in the wrong place :)

NTC (in this PR)